pkg/plugins/runtime/k8s/controllers/configmap_controller.go

```diff
@@ -41,6 +41,10 @@ func (r *ConfigMapReconciler) Reconcile(req kube_ctrl.Request) (kube_ctrl.Result
 		return kube_ctrl.Result{}, nil
 	}
 
+	if err := r.VIPsAllocator.Sync(); err != nil {
```

Comment:
Hmm, why is this one here?

Reply:
`VIPsAllocator#Start` is called only on Universal. On Kubernetes we call `VIPsAllocator#CreateOrUpdateVIPConfig`by reacting on changes in the cluster.